### PR TITLE
fix: core config type validation

### PR DESCRIPTION
**Pull request by sattvikc**
## Summary of change
(A few sentences about this PR)

## Related issues
- Link to issue1 here
- Link to issue1 here

## Test Plan
(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Documentation changes
(If relevant, please create a PR in our [docs repo](https://github.com/supertokens/docs), or create a checklist here highlighting the necessary changes)

## Checklist for important updates
- [ ] Changelog has been updated
- [ ] `pluginInterfaceSupported.json` file has been updated (if needed)
- [ ] Changes to the version if needed
   - In `build.gradle`
- [ ] Had installed and ran the pre-commit hook
- [ ] If there are new dependencies that have been added in `build.gradle`, please make sure to add them in `implementationDependencies.json`.
- [ ] Issue this PR against the latest non released version branch.
   - To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
   - If no such branch exists, then create one from the latest released branch.

## Remaining TODOs for this PR
- [ ] Item1
- [ ] Item2

### CODE REVIEW

**Review by rishabhpoddar — APPROVED** [review]

File: src/main/java/io/supertokens/storage/postgresql/config/Config.java [hunk 1]
```diff
@@ -103,11 +104,16 @@ private PostgreSQLConfig loadPostgreSQLConfig(JsonObject configJson) throws IOEx
...
 
-    public static boolean canBeUsed(JsonObject configJson) {
+    public static boolean canBeUsed(JsonObject configJson) throws InvalidConfigException {
         try {
             final ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
             PostgreSQLConfig config = mapper.readValue(configJson.toString(), PostgreSQLConfig.class);
             return config.getConnectionURI() != null || config.getUser() != null || config.getPassword() != null;
+        } catch (InvalidFormatException e) {
+            throw new InvalidConfigException(
+                    "Cannot set value " + e.getValue().toString() + " for field " + e.getPath().get(0).getFieldName()
```
Comment (rishabhpoddar) [inline src/main/java/io/supertokens/storage/postgresql/config/Config.java:114]: add ' around the config name

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: src/main/java/io/supertokens/storage/postgresql/config/Config.java (hunk 1 above)
Comment (rishabhpoddar) [reply src/main/java/io/supertokens/storage/postgresql/config/Config.java:114]: and around the value.

Comment (rishabhpoddar) [reply src/main/java/io/supertokens/storage/postgresql/config/Config.java:114]: Change the message to `Invalid core config: 'access_token_validity' must be of type long`

Comment (sattvikc) [reply src/main/java/io/supertokens/storage/postgresql/config/Config.java]: Done

**Review by rishabhpoddar — APPROVED** [review]

**Pull request merged**